### PR TITLE
feat(github-comments): add comment reactions column

### PR DESCRIPTION
**Pull request by cathteng**
Add a comment reactions column to `PullRequestComment`. This will make the comment reactions queryable in Redash. The column will only be updated when the comment is less than 30 days old.

### CODE REVIEW

File: src/sentry/models/pullrequest.py
Comment (cathteng) [reply src/sentry/models/pullrequest.py:103]: i'll probably make it null

**Review by AniketDas-Tekky — APPROVED** [review]
ship it after @getsentry/owners-migrations approves

**Review by markstory — APPROVED** [review]

File: src/sentry/models/pullrequest.py
Comment (wedamija) [inline src/sentry/models/pullrequest.py:103]: Is json the right choice here? Should we instead have a table we split out that contains the reactions?

Comment (cathteng) [reply src/sentry/models/pullrequest.py:103]: i'm storing the values from the "reactions" key for instance in this response https://api.github.com/repos/getsentry/sentry/issues/49627

**Pull request merged**
